apps/nowcasting-app/components/helpers/utils.ts

```diff
@@ -573,3 +573,30 @@ export const createBucketObject: (
     increment
   };
 };
+
+export function calculateChartYMax<T extends Record<string, any>>(
```

Comment:
Unless we refactor this to be used in multiple places, let's use the concrete Type for `ChartData` – it might not be set yet so we can handle that, but if it is, it will then be the correct shape/have the right properties for what we need and the tests will be more specific 👍